<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogMusicInstrumentTest</class>
 <widget class="QDialog" name="DialogMusicInstrumentTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>184</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QComboBox" name="comboBoxWavType">
     <item>
      <property name="text">
       <string>Pulse</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Triangle</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sine</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Noise</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="spinBoxFreq">
     <property name="minimum">
      <number>20</number>
     </property>
     <property name="maximum">
      <number>20000</number>
     </property>
     <property name="value">
      <number>432</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="horizontalSliderFreq">
     <property name="toolTip">
      <string>Pitch</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>20000</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Contour</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QSlider" name="verticalSliderT1">
          <property name="toolTip">
           <string>T1</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="verticalSliderT2">
          <property name="toolTip">
           <string>T2</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="verticalSliderT3">
          <property name="toolTip">
           <string>T3</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="verticalSliderMaxVol">
       <property name="toolTip">
        <string>MaxVol</string>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonPlay">
     <property name="text">
      <string>Play</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogMusicInstrumentTest</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogMusicInstrumentTest</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderFreq</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxFreq</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxFreq</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderFreq</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
